region_grower/synthesize_morphologies.py

```diff
@@ -538,13 +565,47 @@ def assign_atlas_data(self, min_depth=25, max_depth=5000):
             )
 
             LOGGER.debug("Extract orientations data for %s region", _region)
-            orientations = self.atlas.orientations.lookup(positions)
+            if self.atlas.orientations is not None:
+                orientations = self.atlas.orientations.lookup(positions)
+            else:
+                orientations = np.array(len(positions) * [np.eye(3)])
             self.cells_data.loc[region_mask, "orientation"] = pd.Series(
                 data=orientations.tolist(),
                 index=self.cells_data.loc[region_mask].index,
                 dtype=object,
             )
 
+            if "directions" in self.atlas.region_structure.get(_region, []):
+                self.cells_data.loc[region_mask, "directions"] = json.dumps(
+                    self.atlas.region_structure[_region]["directions"]
+                )
+            if "boundaries" in self.atlas.region_structure.get(_region, []):
+                boundaries = self.atlas.region_structure[_region]["boundaries"]
+                for boundary in boundaries:
+                    if not Path(boundary["path"]).is_absolute():
+                        boundary["path"] = str(
+                            (self.atlas.region_structure_base_path / boundary["path"]).absolute()
+                        )
+                    if boundary.get("multimesh_mode", "closest") == "territories":
+                        territories = self.atlas.atlas.load_data("glomerular_territories")
```

Comment:
it assumes a specific format of this file, for specific project, so I think we can keep it hidden like that

Reply:
hmmm ok
Normally I would ask to move this specific parts to a specific repo because `region-grower` is supposed to be generic. But it's too much work for the last 2 weeks of BBP :smile:

Reply:
I know, but it would be quite a lot of work to extract these sort of stuff from the code, we could have a mechanism to pass some functions to region-grower, but then we need a was to do it via circuit-build, which is not maintained.. in our next life!

Reply:
Yeah, we can't clean everything before the end anyway. Maybe you can just add some comments for our successors, they will appreciate :)